content/docs/concepts/components/stack.md

```diff
@@ -6,7 +6,48 @@ aliases=[
 ]
 +++
 
-## What is a stack?
+## What is a stack and Why is it deprecated?
+
+### Deprecation
+Stacks are great - long live the stack! 
+However, a stack is basically an extra layer on top of the **build image** (where your app is built)
+and the **run image** (where your app runs) and [the community decided](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)
+to use those two concepts directly without the additional layer formerly known as _stack_. 
```

Comment:
I wonder if it makes sense to compare a stack to a layer?  End-users might expect to find that layer when they run `crane` or some similar tool.

```suggestion
A stack restricts buildpacks to run on top of a set of related **build images** (where your app is built)
and **run images** (where your app runs).  Practical experience of the buildpacks community has not found that the stack concept adds any value therefore [the community decided](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)
to use buildpacks with build and run images without the additional concept formerly known as _stack_. 
```